.github/workflows/changelog.yml

```diff
@@ -0,0 +1,19 @@
+on:
+  pull_request_target:
```

Comment:
Does it need to be `pull_request_target`? On the example for the action they seem to just use a normal `pull_request`.

Reply:
Not sure, it's a verbatim copy from the `stm32f4xx-hal`. 🤷🏻‍♂️

Reply:
Could you try with just pull_request? If that works it's somewhat more straightforward and safer.